### DISCUSSION TITLE
"Smart Timelapse"

### DISCUSSION THREAD

**Discussion by geiseri** [blakeblackshear/frigate#4366] [Ideas]
I recently migrated to frigate from "another platform" and one feature they had that I actually miss is the ability to create a timelapse archive that would run at 2-4x when there was no motion, but slow down to normal speed during the motion clip.  This was handy when I wanted to review things quickly.  I have a way to generate the timelapse from the recording files, but I was not clear on how to figure out which ones were associated with motion events.  I am not sure such a feature makes sense for frigate in general, so I thought it would be a cool addon.  Any hints?

**Comment by NickM-27** [comment]
There are a few existing discussions: https://github.com/blakeblackshear/frigate/issues/609 https://github.com/blakeblackshear/frigate/issues/54

Thought it seems like what you're referring to would have its need filled by a discussed (in other issues) redesign for the recordings page which would include highlights for where motion (1) or object detections (2) occurred

**Comment by geiseri** [comment]
I apologize, I didn't see #609.  To be honest though I was more interested in doing it outside of Frigate.  Things like Zoneminder and Motioneye try to do too much and do it poorly.   I use Home Assistant for the UI in my house.  Things like zwave2mqtt , esphome, and snapcast are awesome in the way they do what they do well and let Home Assistant do what it does well.  Frigate has proven quite the same.   That being said, since all the features are exposed to Home Assistant already it might be a wash.  Either way I like it!

**Comment by geiseri** [comment]
I apologize, I didn't see #609.  To be honest though I was more interested in doing it outside of frigate.  Things like Zoneminder and Motioneye try to do too much, and it suffers.   I use Home Assistant for my House UI.  Things like zwave2mqtt, esphome, and snapcast are awesome the way they do what they do well and let Home Assistant do what it does well.  Frigate has proven quite the same.  =)
